deploy/script/katello-backup

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Echo all commands
+usage(){
+  echo "Usage: katello-backup <backup directory>"
+  echo " eg: $ katello-backup /var/tmp/katello-backup/"
+}
+
+if [ "$#" -ne 1 ]; then
+  echo "ERROR: Please specify an export directory"
+  usage
+  exit 1
+fi
+
+# Backup dir
+umask 0027
+export BDIR=$1
```

Comment:
Is it necessary to export this variable? It's probably not used by anything other than this script? Just `BDIR=$1` is probably all that you need.